src/fal/cli/flow_runner.py

```diff
@@ -95,3 +108,55 @@ def _unique_id_to_model_name(unique_id: str):
 
 def _unique_ids_to_model_names(id_list: List[str]):
     return list(map(lambda id: _unique_id_to_model_name(id), id_list))
+
+
+def _combine_fal_run_results(target_path):
+    last_run_result_file = os.path.join(target_path, "run_results.json")
+    last_run_result = _get_all_result_content(last_run_result_file)
+    last_run_result_array: List[str] = last_run_result["results"]
+    all_results = _distinct_by(
+        reduce(
+            lambda all, next: all.extend(_get_result_array(next)) or all,
+            _get_fal_run_results(target_path),
+            last_run_result_array,
+        ),
+        "unique_id",
+    )
+    last_run_result.pop("results")
+    last_run_result["results"] = all_results
+    os.remove(last_run_result_file)
+    with open(last_run_result_file, "w") as file:
+        file.write(json.dumps(last_run_result))
+        file.close()
+
+
+def _distinct_by(target_list, key):
+    seen_keys = []
+    result = []
+    for item in target_list:
+        if item[key] not in seen_keys:
+            seen_keys.append(item[key])
+            result.append(item)
+    return result
+
+
+def _get_all_result_content(file) -> Dict:
+    with open(file) as content:
+        return json.load(content)
+
+
+def _get_result_array(file) -> List[str]:
+    return _get_all_result_content(file)["results"]
+
+
+def _get_fal_run_results(target_path) -> List[str]:
+    if exists(target_path):
+        fal_results_regex = re.compile("fal_results_\\d*.json")
+        run_result_files = filter(
+            lambda dir: isfile(os.path.join(target_path, dir))
+            and bool(fal_results_regex.match(dir)),
+            os.listdir(target_path),
+        )
+        return list(map(lambda dir: os.path.join(target_path, dir), run_result_files))
+    else:
+        return []
```

Comment:
```suggestion
        run_result_files = filter(lambda p: p.is_file(), Path(target_path).glob("fal_results_*.json"))
        return list(run_result_files)
```

And change return type to `List[Path]` or `map` over the list before returning to with str

```
        return list(map(str, run_result_files))
```

Reply:
👍